technical-reports/format/groups.md

```diff
@@ -58,6 +58,48 @@ The names of items in a group are case sensitive. As per the guidance in the [de
 
 ## Additional group properties
 
+<div class="ednote" title="Group properties vs. nested group and token names">
+  <p>
+    To prevent collisions with token names, token properties are prefixed with a dollar sign (`$`). Using this prefix eliminates the need for a reserved words list and helps future-proof the spec.
+  </p>
+  <p>
+    Group keys without a dollar sign (`$`) prefix denote:
+  </p>
+  <ul>
+    <li>
+      <strong>A token name:</strong> distinguishable by containing a `$value` property
+      <pre>
+      <code>
+{
+  "Group of tokens": {
+    "$description": "This is an example of a group containing a single token",
+    "Token name": {
+      "$value": "#000000"
+    }
+  }
+}
+      </code>
+      </pre>
+    </li>
+    <li>
+      <strong>A nested group name:</strong> distinguishable by <em>not</em> having a `$value` property
+      <pre>
+      <code>
+{
+  "Group of tokens": {
+    "$description": "This is an example of a group containing a nested group",
+    "Subgroup of tokens": {
+      "Token 1 name": { ... },
+      "Token 2 name": { ...}
+    }
+  }
+}
+      </code>
+      </pre>
+    </li>
+  </ul>
+</div>
```

Comment:
Let's use Markdown wherever we can, for readability. (also, valid JSON highlights better)

~~~suggestion
<div class="ednote" title="Group properties vs. nested group and token names">

To prevent collisions with token names, token properties are prefixed with a dollar sign (`$`). Using this prefix eliminates the need for a reserved words list and helps future-proof the spec.

Group keys without a dollar sign (`$`) prefix denote:

- **A token name:** distinguishable by containing a `$value` property

  ```json
  {
    "Group of tokens": {
      "$description": "This is an example of a group containing a single token",
      "Token name": {
        "$value": "#000000"
      }
    }
  }
  ```

- **A nested group name:** distinguishable by _not_ having a `$value` property

  ```json
  {
    "Group of tokens": {
      "$description": "This is an example of a group containing a nested group",
      "Subgroup of tokens": {
        "Token 1 name": {
          "$value": "#aabbcc"
        },
        "Token 2 name": {
          "$value": "#ddeeff"
        }
      }
    }
  }
  ```

</div>
~~~

### Before
<img width="870" alt="Screen Shot 2022-05-10 at 3 33 47 PM" src="https://user-images.githubusercontent.com/85783/167732875-c850e386-d1ab-4b60-8723-2e9fc24ddc13.png">

### After

<img width="870" alt="Screen Shot 2022-05-10 at 3 33 40 PM" src="https://user-images.githubusercontent.com/85783/167732889-5971e8da-9c0c-4cd5-9ce0-09969aea43f2.png">